### PR TITLE
2 update

### PR DESCRIPTION
**Pull request by rmchayes**
Please check

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
         <title>Example</title>
     </head>
     <body>
-		<h1>welcome to my nightmare</h1>
+		<h1>welcome to my nightmare!</h1>
```
Comment (rmchwork) [inline index.html]: Think you meant two exclamation points ??

**Review by rmchwork — COMMENTED** [review]
Please see my comment about !!

File: index.html (hunk 1 above)
Comment (rmchayes) [reply index.html]: Oops!!

**Review by rmchwork — CHANGES_REQUESTED** [review]
As discussed !!

File: index.html [hunk 2]
```diff
@@ -4,6 +4,8 @@
         <title>Example</title>
     </head>
     <body>
+		<h1>welcome to my nightmare!!</h1>
         <p>This is an example of a simple HTML page with one paragraph.</p>
     </body>
 </html>
+"
```
Comment (rmchwork) [inline index.html]: Why is this here??

**Review by rmchwork — CHANGES_REQUESTED** [review]
Think you made a mistake...

File: index.html (hunk 2 above)
Comment (rmchayes) [reply index.html]: Sorry...

Comment (rmchayes) [reply index.html]: Will fix

**Review by rmchwork — APPROVED** [review]
Ok you can merge it...
